beetmoverscript/utils.py

```diff
@@ -65,22 +65,28 @@ def generate_beetmover_template_args(task, release_props):
         "platform": release_props["platform"],
     }
 
+    if cert != 'nightly':
+        template_args["build_number"] = release_props["buildnumber"]
```

Comment:
I meant to ask this before. Will we have the build number coming from somewhere within the `balrog_props` or we need to find another way to pass that?